Docker/docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+version: '2'
+services:
+  mariadb:
+    image: docker.io/bitnami/mariadb:10.4
+    environment:
+      # ALLOW_EMPTY_PASSWORD is recommended only for development.
+      - MARIADB_ROOT_PASSWORD=test1234
+      - MARIADB_PASSWORD=test1234
+      - MARIADB_USER=bn_magento
+      - MARIADB_DATABASE=zip_magento
+    volumes:
+      - "./mariadb-data:/bitnami/mariadb"
+  magento:
+    image: bitnami/magento:${MAGENTO_TAG}
+    ports:
+      - '80:8080'
+      - '443:8443'
+    environment:
+      - MAGENTO_HOST=localhost
+      - MAGENTO_DATABASE_HOST=mariadb
+      - MAGENTO_DATABASE_PORT_NUMBER=3306
+      - MAGENTO_DATABASE_USER=bn_magento
+      - MAGENTO_DATABASE_NAME=zip_magento
+      - MAGENTO_DATABASE_PASSWORD=test1234
+      - MAGENTO_USERNAME=admin
+      - MAGENTO_PASSWORD=P@ssw0rd
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: High

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |

 <!-- 3ab76a81-fef2-4d70-87e4-723fbd893871 -->